CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -13,9 +13,10 @@ private typealias Section = Styleguide.Section
 public struct MyCoinView: View {
 
     @EnvironmentObject var myCoinState: MyCoinState
+    @State private var selectedTimeFrameItem: TimeFrameItem = .init(timeFrame: .oneDay)
 
     public var body: some View {
-        List {
+        ScrollView {
```

Comment:
I don't know if this is Apple's intention or a bug, but it seems that by default the `List` _observes_ or _handles_ tapping views inside of it. So when our view, which is the `TimeFrameSection` has a button with action, it gets confused when we tap.

Try to keep this using `List`, but in the `TimeFrameSection` add this line:
```swift
ForEach(timeFrameList) { item in
  Button(item.title) {
    selected = TimeFrameItem(timeFrame: item.timeFrame)
  }
  .buttonStyle(.borderless) 👈  add this
  // ...
}
```

Reply:
Yeah, the behavior's so weird :D. The solution is excellent 🙏 . I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/99/commits/2fab7d69ff3151a83b4dd512405f946144981c0d. Thank you.